### ISSUE TITLE
Error during setup-appliance.sh: 

### ISSUE THREAD

**Issue by johanneskastl** [openSUSE/open-build-service#14702]
# Issue Description
When running `setup-appliance.sh` there are two errors due to missing executables (or permissions).

```
/usr/lib/obs/server/setup-appliance.sh: line 656: ./obssrcserver: No such file or directory                                                                                                  
/usr/lib/obs/server/setup-appliance.sh: line 657: ./obsrepserver: No such file or directory
```

Expected Result
============
Those errors should not appear.

How to Reproduce
===============
1. Install OBS via OBS_Server pattern on Leap 15.4
2. Run `/usr/lib/obs/server/setup-appliance.sh`
3. Check the output

Further Information
===============

```
$ rpm -qf /usr/lib/obs/server/setup-appliance.sh
obs-common-2.10.20-lp154.109.1.noarch
```

**Comment by johanneskastl** [comment]
I think this comes from here:

```
  GPG_KEY_CREATED=0

  prepare_obssigner

  if [[ $GPG_KEY_CREATED == 1 ]];then
    pushd .
    # avoid systemctl
    cd /etc/init.d
    ./obssrcserver reload
    ./obsrepserver reload
    popd
  fi
```

**Comment by pallaswept** [comment]
Yep, the current script assumes the presence of init scripts which it does not install so they do not exist. Systemd `requires` directives will ensure they load up after a restart, but you can start them manually to ensure the installation completes successfully, by using the same start command that systemd would. I can't find any documentation as to why systemctl is being avoided so it might be perfectly safe to just start them with systemctl but since the script specifically avoids it, so do I.
